### PR TITLE
Add cards to landing page

### PR DESCRIPTION
**Pull request by melissawm**
Includes a few fixes to content and looks.

For the tutorials landing page, we need to know how we want to group the tutorials so the cards can point to specific sections.

Addresses #21

### CODE REVIEW

**Comment by melissawm** [comment]
This is how it looks like:

Screenshot_20231130_161843

**Review by aliddell — APPROVED** [review]
Cards look good to me. I supplied the link to the multiscale tutorial in a suggestion.

**Review by dgmccart — APPROVED** [review]

File: docs/tutorials/compressed.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ import zarr
 compressed = zarr.open(config.video[0].storage.settings.filename)
 ```
 
-We'll print some of the data properties to illustrate how the data was compressed. Since we have not enabled [multiscale](link to tutorial?) output, `out.zarr` will only have one top level array`"0"`.
+We'll print some of the data properties to illustrate how the data was compressed. Since we have not enabled [multiscale]() output, `out.zarr` will only have one top level array`"0"`.
```
[the remark this answers is not in the document]
Comment (dgmccart) [reply docs/tutorials/compressed.md]: Whoops, we were reviewing at the same time. I also added a link.

**Comment by melissawm** [comment]
Thanks!

**Pull request merged**